### PR TITLE
Rewrite script without jQuery

### PR DESCRIPTION
**Pull request by kyoshino**
Refactor `releasehealth.js` using ECMAScript 201x. Goodbye jQuery!

Try it: https://kyoshino.github.io/releasehealth/

### CODE REVIEW

File: js/releasehealth.js [hunk 1]
```diff
@@ -4,105 +4,125 @@
...
-  // Display title
-  $("#title").text(versions[channel].title + " " + versions[channel].version);
-
-  displayMeasures();
-
-  // if there is a display=bigscreen parameter in the url, we hide some clutter (office TV displays)
-  if (getURLParam('display') === 'bigscreen') {
-    document.body.classList.add("bigscreen");
+/**
+ * Impalement the Firefox Release Health Dashboard.
```
Comment (emceeaich) [inline js/releasehealth.js]: s/Impalement/Implement/

**Comment by kyoshino** [comment]
The padding change is apparently from #44 (Why it’s not deployed yet?) but I can fix that.

**Comment by kyoshino** [comment]
Maybe the style could be further improved, but the numbers are now vertically centred for a better look.

**Comment by emceeaich** [comment]
Changes didn't show up because this repo still uses the gh-pages branch to publish to web. I needed to merge @kyoshino and @pascalchevrel's changes into gh-pages.

**Pull request merged**